### PR TITLE
Add default yaml config options

### PR DESCRIPTION
**Pull request by jun2zhou**

### CODE REVIEW

File: chipsec/utilcmd/smbios_cmd.py [hunk 1]
```diff
@@ -32,20 +32,27 @@
...
     def requirements(self) -> toLoad:
         return toLoad.All
 
     def parse_arguments(self) -> None:
+        options = Options()
+        util_cfg = options.get_section_data('Util_Config')
+        try:
+            default_type = util_cfg['smbios_get_type']
+        except:
+            default_type = None
```
Comment (npmitche) [inline chipsec/utilcmd/smbios_cmd.py:48]: Same with this. This should default to "raw"

File: chipsec/options.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Contact information:
+# chipsec@intel.com
+#
+
+import os
+import yaml
```
Comment (npmitche) [inline chipsec/options.py:22]: Open question: Do we want to use YAML and introduce a new dependency or do we want to use JSON (or similar) which we already use other places and don't need to introduce a new dependency?  
@BrentHoltsclaw  @frinzell

Comment (BrentHoltsclaw) [reply chipsec/options.py:22]: YAML is much easier to read and edit than JSON.  If there is a concern about adding a dependency then should probably look into using a configuration file with configparser

Comment (frinzell) [reply chipsec/options.py:22]: Small YAML files are fine for reading, larger ones not so much.  The indention-based nesting is the worst, IMO.  Would rather use TOML.  But JSON or even XML would be preferred over YAML.

**Pull request closed without merging**